python/ray/autoscaler/command_runner.py

```diff
@@ -45,7 +47,12 @@ def __init__(self,
 def _with_interactive(cmd):
     force_interactive = ("true && source ~/.bashrc && "
                          "export OMP_NUM_THREADS=1 PYTHONWARNINGS=ignore && ")
-    return ["bash", "--login", "-c", "-i", quote(force_interactive + cmd)]
+    hide_ttys = " 2> >(tee 1>&2) | tee"
+    # todo: this might need to be disabled iff we are dumping output
+
+    return [
+        "bash", "--login", "-c", "-i",
```

Comment:
This won't work on Windows either, though I guess it was already there.

Reply:
Yeah I didn't touch this :)